src/assets/scss/_global.scss

```diff
@@ -1,3 +1,10 @@
+@font-face {
+  font-family: 'gibsonlight';
```

Comment:
This is coming from a place of ignorance; do many email clients support `@font-face`?